### DISCUSSION TITLE
Frigate in a business environment

### DISCUSSION THREAD

**Discussion by nckrwlmn** [blakeblackshear/frigate#15165] [Ask A Question]
I am looking to set up frigate to monitor a business, not a home. I see there's lots of emphasis on home assistant integration and the like. I am hoping to migrate from synology surveillance to frigate, hopefully because the detection will be better and the live views will be more responsive. What are the recommendations to try to mimic SS as far as it having a native app in App Store, having multiple users being able to view footage and share, and receiving notifications when something happens. I'm using pushover but I'm thinking these notifications would get lost in other notifications. I don't think home assistant scales to many users, it wasn't made to. Pushover could probably scale, but I have yet to see if it's a useful notification, maybe even with a photo attached. What would you recommend?

**Comment by NickM-27** [comment]
Frigate supports being installed as  PWA (progressive web app) so it would work just like an app on the app store. Also, Frigate 0.15 (currently in beta) supports notifications directly

**Reply by nckrwlmn** [reply]
Thank you very much for the info that's wonderful. I will follow up once I actually install this thing.
